platform/src/test/kotlin/org/stellar/anchor/platform/callback/RestRateIntegrationTest.kt

```diff
@@ -0,0 +1,231 @@
+package org.stellar.anchor.platform.callback
+
+import com.google.gson.Gson
+import io.mockk.every
+import io.mockk.mockk
+import java.time.Instant
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullSource
+import org.junit.jupiter.params.provider.ValueSource
+import org.stellar.anchor.api.callback.GetRateRequest
+import org.stellar.anchor.api.callback.GetRateRequest.Type.from
+import org.stellar.anchor.api.callback.GetRateResponse
+import org.stellar.anchor.api.exception.ServerErrorException
+import org.stellar.anchor.api.sep.AssetInfo
+import org.stellar.anchor.api.sep.AssetInfo.Schema.iso4217
+import org.stellar.anchor.api.sep.AssetInfo.Schema.stellar
+import org.stellar.anchor.asset.AssetService
+import org.stellar.anchor.util.GsonUtils
+
+class RestRateIntegrationTest {
```

Comment:
Can you please add tests here for assets that have different scale?
The best example is (fiat) with scale 2 <-> USDC with scale 7 conversion

Reply:
Changed to `usdcAssetInfo.significantDecimals = 7`